packages/server/graphql/mutations/downgradeToPersonal.ts

```diff
@@ -41,7 +41,8 @@ export default {
 
     // RESOLUTION
     // if they downgrade & are re-upgrading, they'll already have a stripeId
-    const {teamIds} = await resolveDowngradeToPersonal(orgId, stripeSubscriptionId!, viewerId)
+    await resolveDowngradeToPersonal(orgId, stripeSubscriptionId!, viewerId)
+    const {teamIds} = await dataLoader.get('teamsByOrgId').load(orgId)
```

Comment:
this code was not tested! this dataloader will not return an object containing `teamIds`.
can you please:
- add a `GQLContext` type to the context object at the top so we get some type safety
- add a checklist of tests that you performed (see https://github.com/ParabolInc/parabol/pull/5205 as an example)